src/main/resources/application-preprod.yaml

```diff
@@ -52,29 +50,29 @@ no.nav.security.jwt:
           client-auth-method: private_key_jwt
           client-jwk: ${TOKEN_X_PRIVATE_JWK}
 
-
-
+funksjonsbrytere:
+  kafka.enabled: false
```

Comment:
Tenker vi må ha en prat om hvordan vi skal svitsje kafka, men at det er greit å starte med kafka av. Skriver likevel ned noen punkter.

- Siden de har forskjellig group id så kan det i prinsippet være 2 apper som lytter på samme topic.
- For de topicene med EARLIEST offset i config, så vil man spole igjennom alle meldinger første gang. Dette gjelder pdl leesah og EF-topicene.
- De med latest offset strategy vil starte med den siste og er bare å svitsje over når man er klar
- For de med EARLIEST strategy så bør man enten spole igjennom alle meldinger slik at de havner i hendelse-tabellen, med koden som gjør endringer kommentert ut eller back en toggle som er skrudd av, eller midlertidig endre til latest.